.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -36,17 +36,17 @@ jobs:
           - {os: "ubuntu-20.04", python-version: "3.8", pytorch-version: "1.10"}  # ex-conda
           - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.11"}  # ex-conda
           - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.12"}  # ex-conda
-          - {os: "ubuntu-20.04", python-version: "3.10", pytorch-version: "1.12"}
+          - {os: "ubuntu-20.04", python-version: "3.10", pytorch-version: "1.12", draft: "true"}
           - {os: "macos-11", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
```

Comment:
@akihironitta unrelated question. why do we not set `requires` for all jobs?

Reply:
@carmocca I'm sorry I missed your message here. `requires: "latest"` wasn't used in any steps' conditions in the workflow, so I only kept `requires: "oldest"`. We can add those back here (or remove them in other workflows, too) for consistency.

Reply:
It's okay. Now I see that it's simpler to omit it